resources/views/auth/login.blade.php

```diff
@@ -38,10 +38,11 @@ class="flex flex-col px-4 py-8 mx-4 border-2 rounded-xl sm:px-8 border-theme-sec
                     @php
                         $username = \Laravel\Fortify\Fortify::username();
                         $usernameAlt = Config::get('fortify.username_alt');
+                        $email = Config::get('fortify.email');
                         $type = 'text';
 
-                        if ($usernameAlt) {
-                            $label = trans('fortify::forms.'.$username).' or '.trans('fortify::forms.'.$usernameAlt);
+                        if ($usernameAlt === $username && ! is_null($email)) {
```

Comment:
I don't fully understand the logic of this and it's risky to make this `if` very specific without handling the previous scenario.

Reply:
why is `usernameAlt` even the same as `username`? shouldn't you simply omit the `usernameAlt` setting if you only need 1 option to logins? It makes little sense to me to set both to the same property to force a single login method; feels hacky